src/server/scripts/Spells/spell_hunter.cpp

```diff
@@ -691,27 +691,54 @@ class spell_hun_masters_call : public SpellScriptLoader
             return ValidateSpellInfo({ SPELL_HUNTER_MASTERS_CALL_TRIGGERED });
         }
 
+        SpellCastResult DoCheckCast()
+        {
+            Guardian* pet = GetCaster()->ToPlayer()->GetGuardianPet();
+            ASSERT(pet); // checked in Spell::CheckCast
```

Comment:
why do we need to crash the server here? ...also consider to fix this first: https://github.com/azerothcore/azerothcore-wotlk/issues/5178 otherwise it could crash easily

Reply:
I will investigate about that